src/index.html

```diff
@@ -5,8 +5,389 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com/%22%3E%22%3E">
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E%22%3E">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__img">
+        <img
+          class="img_item"
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_item__is-active is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="nav-hover"
+            >Ноутбуки и компьютеры</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Гаджеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Планшеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a  href="#" class="nav-link">Фото</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="nav-link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
```

Comment:
```suggestion
    <main>
       <div class="cards">
          <div class="card cards__container" data-qa="card">
       </div>
```
Я бы лучше создал отдельный класс для всех карточек)